src/main/java/mate/academy/bookshop/specification/book/impl/TitleSpecification.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.bookshop.specification.book.impl;
+
+import java.util.Arrays;
+import mate.academy.bookshop.dto.BookSearchParameters;
+import mate.academy.bookshop.model.Book;
+import mate.academy.bookshop.specification.book.BookSpecificationProvider;
+import org.springframework.data.jpa.domain.Specification;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TitleSpecification implements BookSpecificationProvider {

```

Comment:
same for other similar classes
```suggestion
public class TitleSpecificationProvider implements BookSpecificationProvider {
```